src/bp-templates/bp-nouveau/includes/template-tags.php

```diff
@@ -1333,6 +1339,10 @@ function bp_nouveau_get_nav_count() {
 		$count = bp_media_get_total_group_media_count();
 	} elseif ( 'groups' === $bp_nouveau->displayed_nav && bp_is_active( 'media' ) && bp_is_group_albums_support_enabled() && 'albums' === $nav_item->slug ) {
 		$count = bp_media_get_total_group_album_count();
+	} elseif ( 'groups' === $bp_nouveau->displayed_nav && $coursesSlug === $nav_item->slug ) {
```

Comment:
@shohelbuddyboss Can you follow the naming convention of the variable name of WP `coursesSlug` should be a `courses_slug` ? Can you make the change please? and also make the tabbing proper here?